app/assets/javascripts/participants.js

```diff
@@ -1,12 +1,11 @@
-$(document).ready(function(){
+$(function(){
   if($('.participants .uniqueness-warning').is(':visible'))
     $('.uniqueness-warning').hide();
 
   $('.participants').on('change', 'input[type=number]', function(){
     var existingNumbers = $('.participants input[type=number]').not($(this)).map(function(){
       return parseInt($(this).val());
     });
-    existingNumbers = $.unique(existingNumbers);
```

Comment:
This is deprecated, but also, an array doesn't need to be sorted or unique in order to check it for the presence of an element.